src/index.html

```diff
@@ -1,16 +1,59 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="container">
+
+      <div class="wrapper">
+
+        <form action="https://google.com" data-qa="keypress" method="get">
+          <label for="big">
+            <input
+            type="search"
+            placeholder="Try ''Los Angeles''"
+            data-qa="big"
+            class="searchbar_big">
+            <img
+             src="./images/Search.svg"
+             alt="google"
+             class="searchbar_img_big">
```

Comment:
Go through the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) and fix your code style, look closely at the 3rd item